server/controllers/payment-controller.js

```diff
@@ -0,0 +1,32 @@
+const User = require("../models/user-model.js");
+const Course = require("../models/course-model.js");
+
+//@desc Buy course
+//@route Get api/payment/buycourse
```

Comment:
Please use ALL UPPERCASE for HTTP request methods.
Also, should we need the route `api/payment/buycourse/`? Or just `api/payment` or `api/payment/process` is fine.